README.md

```diff
@@ -71,5 +71,14 @@ While the sample loader pod is running, from another shell run
 kubectl cp <path_to_sample_file> <namespace>/<sample_load_pod_name>:<destination_path_on_pod>
 ```
 
+### Validating a sample file
```

Comment:
Should be a `##` level heading at the end of the file I think, since the `Logging` section refers to the sample loader.